packages/run-protocol/src/vaultFactory/types.js

```diff
@@ -234,39 +211,20 @@
 
 /**
  * @typedef {Object} VaultParamManager
- * @property {GetParams} getParams
+ * @property {() => Record<Keyword,ParamShortDescription> & {
+ *  'InitialMargin': ParamRecord<'ratio'> & { value: Ratio },
+ *  'InterestRate': ParamRecord<'ratio'> & { value: Ratio },
+ *  'LiquidationMargin': ParamRecord<'ratio'> & { value: Ratio },
+ *  'LoanFee': ParamRecord<'ratio'> & { value: Ratio },
+ * }} getParams
  * @property {(name: string) => bigint} getNat
  * @property {(name: string) => Ratio} getRatio
- * @property {(period: bigint) => void} updateChargingPeriod
- * @property {(period: bigint) => void} updateRecordingPeriod
  * @property {(margin: Ratio) => void} updateInitialMargin
  * @property {(margin: Ratio) => void} updateLiquidationMargin
  * @property {(ratio: Ratio) => void} updateInterestRate
  * @property {(ratio: Ratio) => void} updateLoanFee
  */
 
-/**
- * @typedef {Object} ElectorateParamManager
- * @property {GetParams} getParams
- * @property {(name: string) => Amount} getInvitationAmount
- * @property {(name: string) => Invitation} getInternalParamValue
- * @property {(invitation: Invitation) => void} updateElectorate
- */
-
-/**
- * @callback MakeVaultParamManager
```

Comment:
Good riddance to the use of `@callback` for something that isn't actually passed around as a callback.